storage/aptosdb/src/schema/state_value_index/mod.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::schema::{ensure_slice_len_eq, ensure_slice_len_gt, STATE_VALUE_INDEX_CF_NAME};
+use anyhow::Result;
+use aptos_schemadb::{
+    define_schema,
+    schema::{KeyCodec, SeekKeyCodec, ValueCodec},
+};
+use aptos_types::{
+    state_store::{state_key::StateKey, state_key_prefix::StateKeyPrefix},
+    transaction::Version,
+};
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use std::{io::Write, mem::size_of};
+
```

Comment:
could we add a brief description on the motivation of having a new schema?

Reply:
Added in the PR description.

Reply:
maybe add it to the top of code file as well ?

Reply:
Done